src/pykiso/lib/auxiliaries/udsaux/uds_auxiliary.py

```diff
@@ -0,0 +1,384 @@
+##########################################################################
+# Copyright (c) 2010-2022 Robert Bosch GmbH
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0.
+#
+# SPDX-License-Identifier: EPL-2.0
+##########################################################################
+
+"""
+uds_auxiliary
+*************
+
+:module: uds_auxiliary
+
+:synopsis: Auxiliary used to handle Unified Diagnostic Service protocol
+
+.. currentmodule:: uds_auxiliary
+
+"""
+import configparser
+import logging
+import threading
+from pathlib import Path
+from typing import List, Optional, Union
+
+import can
+
+from pykiso.connector import CChannel
+from pykiso.interfaces.thread_auxiliary import AuxiliaryInterface
+
+from . import uds_exceptions
+from .uds_request import UDSCommands
+from .uds_response import UdsResponse
+from .uds_utils import get_uds_service
+
+log = logging.getLogger(__name__)
+
+Uds = None
+createUdsConnection = None
+IsoServices = None
+
+
+class UdsAuxiliary(AuxiliaryInterface):
+    """Auxiliary used to handle UDS messages"""
+
+    NEGATIVE_RESPONSE = 0x7F
```

Comment:
Why do we have here the negative response defined when it is already defined in uds_response.py?

Reply:
I jsut split the UDS aux nothing else